### ISSUE TITLE
SelectOneRadio: with custom layout keyboard navigation does not work properly

### ISSUE THREAD

**Issue by OBreidenbach** [primefaces/primefaces#5503]
## 1) Environment
- PrimeFaces version: Showcase ->PrimeFaces-8.0-SNAPSHOT
- Does it work on the newest released PrimeFaces version? Version? no
- Does it work on the newest sources in GitHub? (Build by source -> https://github.com/primefaces/primefaces/wiki/Building-From-Source) no
- Application server + version:
- Affected browsers: FF, Chrome, ...

## 2) Expected behavior
In custom layout the radios are reachable by the keyboard tab as with other layout.

## 3) Actual behavior
After a radio in the custom layout is selected, the keyboard tab is ignoring the radios of this group.

## 4) Steps to reproduce
Got to the showcase: https://www.primefaces.org/showcase/ui/input/oneRadio.xhtml
1. Click on the radio at 'Responsive' - Amsterdam
1. Use the tab key to go to the following radios of the 'Custom Layout'
1. As long as you do not select any of thus radios it is working fine.
1. Select one of the radios of the custom layout
1. After selected one of the radios, 'Custom Layout' radios are ignored on tab key access.

## 5) Sample XHTML
Not provided. See showcase: https://www.primefaces.org/showcase/ui/input/oneRadio.xhtml

## 6) Sample bean
Not provided. See showcase: https://www.primefaces.org/showcase/ui/input/oneRadio.xhtml

**Comment by melloware** [comment]
I will admit i thought this was going to be easy to fix but its actually quite tricky.

**Comment by jepsar** [comment]
The fix for #8180 makes fixing this easier (I hope).

**Comment by melloware** [comment]
Let me know if it does. I was stumped on this one before!

**Comment by jepsar** [comment]
I'll have a look when the PR for #8180 is merged.

**Comment by melloware** [comment]
I just tested this after the latest changes and it looks like the Tab key reaches the first radio in a component but subsequent tabs do not tab through the radios it goes to the next whole component.

**Comment by jepsar** [comment]
I did not spent time on that yet. I didn't want the other PR to get to big or complex.

**Comment by melloware** [comment]
No worries figured I would just test this and update.

**Comment by jepsar** [comment]
Got this one figured out. We need to NOT `display: none` the original inputs (prevents them from getting focus) and we need listeners on them to transfer the focus to the custom radio buttons. Will fix it tonight and I think I'd like to add an integration test here.

**Comment by jepsar** [comment]
The issue is worse, keyboard navigation on regular select one radio's doesn't work at all with Safari.

**Comment by jepsar** [comment]
What caused the weird behavior of first being able to select a button and no longer once a button was selected was caused by part of the buttons in the radio group were hidden. The only thing that had to change was having a distinct group name for the custom buttons, so that they won't interfere with the original, hidden radio buttons.
